content/docs/howto/create-custom-stack.md

```diff
@@ -56,30 +59,140 @@ repo to view the Dockerfiles we have defined for both the base image and CNB ima
 3. Build the stack image with  `docker build . -t <stack-name>-<run or
    build>:<tag> --target <target>` for both the build and run images.
 4. Push the stack images to a registry with `docker push`
-5. Create a custom builder with the stack you want to use. Check out the [builder
-   documentation][concepts/builders] for details on builders. This can be
-   achieved by cloning the builder you want to use, and modifying the
-   `builder.toml` file. For example, if you have built a custom stack based off
-   of the Paketo Full stack, you will want to add it to the [Full
+
+
+### Using `jam create-stack`
+
+#### Prerequisites
+You will need the following tools installed on your machine:
+<!-- spellchecker-disable -->
+- [jam](https://github.com/paketo-buildpacks/jam)
+- [skopeo](https://github.com/containers/skopeo)
+<!-- spellchecker-enable-->
+
+1. Create a Dockerfile for the build and run stack images, as in steps 1 and 2 above.
+
+2. Create a `stack.toml`, which should resemble the following:
+
+<!-- spellchecker-disable -->
+
+{{< code/copyable >}}
+id = "io.paketo.stacks.tiny"
+
+platforms = ["linux/amd64"]
+
+[build] 
+  dockerfile = "<path/to/build/Dockerfile>"
+  gid = 1000
+  shell = "/bin/bash"
+  uid = 1000
+
+  [build.args]
+    sources = """
+    deb http://archive.ubuntu.com/ubuntu bionic main universe multiverse
+    deb http://archive.ubuntu.com/ubuntu bionic-updates main universe multiverse
+    deb http://archive.ubuntu.com/ubuntu bionic-security main universe multiverse
+    """
+
+    # List of packages which should be included in the stack build image
+    packages = """\
+    <some-package> \
+    <another-package> \
+    """
+
+[run]
+  dockerfile = "<path/to/run/Dockerfile>"
+  gid = 1000
+  shell = "/sbin/nologin"
+  uid = 1000
+
+  [run.args]
+    sources = """
+    deb http://archive.ubuntu.com/ubuntu bionic main universe multiverse
+    deb http://archive.ubuntu.com/ubuntu bionic-updates main universe multiverse
+    deb http://archive.ubuntu.com/ubuntu bionic-security main universe multiverse
+    """
+
+    # List of packages which should be included in the stack run image
+    packages = """\
+    <some-package> \
+    <another-package> \
+    """
+
+[deprecated]
+  legacy-sbom = true
+  mixins = true
+{{< /code/copyable >}}
+
+<!-- spellchecker-enable-->
+
+
+
+3. Create the stack with the `jam` CLI:
+<!-- spellchecker-disable -->
+{{< code/copyable >}}
+jam create-stack --config stack.toml --build-output <name>.oci --run-output <name>.oci
+{{< /code/copyable >}}
+<!-- spellchecker-enable-->
+
+
+4. Use `skopeo` to copy the OCI archives
+   to the desired registry:
+
+   To copy the archives to a remote registry:
+<!-- spellchecker-disable -->
+{{< code/copyable >}}
+skopeo copy oci-archive:///<path/to/oci/archive/> docker://<registry-image-location>:<tag>
+{{< /code/copyable >}}
+
+   **Example:**
+
+   `skopeo copy oci-archive:///workspace/tiny-stack/build.oci docker://index.docker.io/tiny-build:latest`
```

Comment:
This creates too many lines on mobile. I don't think we need to provide an example because the copyable code is straight forward.

Reply:
Agreed - I think the two existing code-snippets (copying to remote registry and copying to local docker deamon) are sufficient.